ui/src/components/App.spec.js

```diff
@@ -19,11 +22,16 @@ const setup = (propOverrides) => {
   };
 };
 
+/**
+ * @todo get these tests working
+ */
 describe('components', () => {
   describe('App', () => {
     it('should render', () => {
-      const { output } = setup();
-      expect(output.type).toBe(Router);
+      const { output } = setup({ isLoggedIn: true, isLoading: true });
+
+      const app = output.props.children;
+      expect(app.type).toBe(undefined);
```

Comment:
```suggestion
      expect(app.type).toBe(Router);
```
Once you export `App` without router & import it above as suggested, you should be able to test the component renders a `Router` object as per the render method